docs/reference-guides/richtext.md

```diff
@@ -110,13 +110,6 @@ wp.blocks.registerBlockType( /* ... */, {
 
 While using the RichText component a number of common issues tend to appear.
 
-### Placeholder Content Separates from the Input
-
-In some cases the placeholder content on RichText can appear separate from the input where you would write your content. This is likely due to one of two reasons:
-
-1. You can't have an element with the CSS `display` property set to `inline`. You will need to set it to `inline-block` or any other value.
-2. The `position` CSS property value for the element must be set to `relative` or `absolute` within the admin. If the styles within style.css or editor.css modify the `position` property value for this element, you may see issues with how it displays.
```

Comment:
This is no longer relevant either. The placeholder is purely text and no longer positioned over the rich text field.